SimplenoteIntents/SimplenoteIntents.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>group.com.codality.NotationalFlow.Alpha</string>
```

Comment:
I think this should be `group.$(CFBundleIdentifier)` instead.

Reply:
cc @oguzkocer OR @mokagio OR @jkmassel  just in case!!

Reply:
Nice suggestion @jleandroperez.

The result at compile-time or (run-time 🤔 ?) would be the same as this syntax, because `group.$(CFBundleIndentifier)` resolves to the bundle identifier value of the target and build configuration combination.

Having said that, it would be useful to use this syntax because it would allow us to condense `.entitlements` files that are otherwise identical. E.g.:

https://github.com/Automattic/simplenote-ios/blob/08d15d7240a7e8d23544708969451bf6f6ff4815/SimplenoteIntents/SimplenoteIntentsRelease.entitlements#L1-L10
and 

https://github.com/Automattic/simplenote-ios/blob/08d15d7240a7e8d23544708969451bf6f6ff4815/SimplenoteIntents/SimplenoteIntentsDistribution%20Internal.entitlements#L1-L10

I think my suggestion is more appropriate for a dedicated PR, though.

Reply:
I opened a PR against this branch to address an entitlements issue I noticed while researching a good answer for the question above: https://github.com/Automattic/simplenote-ios/pull/1408